src/entities/Place/model.ts

```diff
@@ -139,6 +139,32 @@ export default class PlaceModel extends Model<PlaceAttributes> {
     return queryResult[0]
   }
 
+  static async findByIds(placeIds: string[]): Promise<PlaceAttributes[]> {
+    const sql = SQL`
+      SELECT p.*
+      FROM ${table(this)} p
+      WHERE "p"."id" IN ${values(placeIds)}
+    `
+
+    const queryResult = await this.namedQuery("find_by_ids", sql)
+    return queryResult
+  }
+
+  static async countByIds(placeIds: string[]) {
+    const query = SQL`
+      SELECT
+        count(DISTINCT p.id) as "total"
```

Comment:
We should replicate the same query as the last one to get a consistent value on the total.
Could there be places where the id is the same as others?